.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
Why allow the blank issue?

Reply:
I misunderstood the meaning of this option, my false. We can change this setting to "false"

Reply:
Is it required for the APISIX discussion forum?

Reply:
Yes. We want all support questions to go to Discussions. But people are used to create issues. It will take long time until everyone learns the new feature. Having a big green button in the issue creation seems like a good idea.
![image](https://user-images.githubusercontent.com/47442074/129675445-0f81052c-a175-45d4-9434-4f1eec3f3468.png)

Reply:
Actually, I mean whether "blank_issues_enabled" is required for the APISIX discussion forum. Allow blank issues or not seems to be irrelative to the discussion forum.

Reply:
We can encourage contributors to use issue templates by setting blank_issues_enabled to false. If we set blank_issues_enabled to true, people will have the option to open a blank issue.
Do you think it is necessary?

Reply:
Agree